src/InstagramClone.Api/Controllers/UsersController.cs

```diff
@@ -61,9 +58,54 @@ public IActionResult CreateUsers([FromBody] UserCreateDTO userDto)
             this._context.SaveChanges();
 
             // returns a url of the user that was created 
+            // https://www.code4it.dev/blog/createdAtRoute-createdAtAction
             // https://stackoverflow.com/questions/47939945/how-to-use-created-or-createdataction-createdatroute-in-an-asp-net-core-api#64315534
             return CreatedAtAction(nameof(GetUser),new {id = userDto.Id },  userDto);
         } 
 
+        [HttpPost("follow")]
+        public IActionResult Follow([FromBody]FollowersDto followersDto)
+        {
+
+            // "helped me alot" https://www.youtube.com/watch?v=j1e6Z-7QNpk
+
+            var followed = followersDto.FollowedUserId; // 
+            var following = followersDto.FollowerId;
+
+            var uuF = _context.Users.Find(followersDto.FollowedUserId);
+            var uF = _context.Users.Find(followersDto.FollowerId);
+            var followersTofollow = new UserFollower()
+              {
+                  FollowerId = followersDto.FollowerId,
+                  FollowedUserId = followersDto.FollowedUserId,
```

Comment:
```suggestion
                  FollowedUserId = followersDto.FollowedUserId,
                  CreatedDate = DateTime.Now()
```

It's important that we save the date and time that the user was followed by the other user